lib/config/schemas/system.config.json

```diff
@@ -10,7 +10,7 @@
           "properties": {
             "emulate": {
               "type": "boolean",
-              "default": "true"
+              "default": false
```

Comment:
why ?

Reply:
`"true"` is a string, not a boolean. `ajv` is not configured to coerce values.

Also, I had to change the default value from `true` to `false` because if not, the following scenario would not work:


```yml
db:
  redis:
    host: redis
    port: 6379
    namespace: EG
```

Because `emulate` would default to `true`, thous skipping my connection parameters.

Reply:
ah, ok. As long as default templates have emulate:true set explicitly it is ok

Reply:
Yes, they have it. I've checked them.